pkg/kubectl/cmd/cmd.go

```
@@ -347,6 +347,21 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 			},
 		},
 	}
+
+	// Loads plugins and create commands for each plugin identified
+	loadedPlugins, _ := f.PluginLoader().Load()
```

Don't swallow the error, log it, please.